src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -556,6 +555,63 @@ def test_containerapp_ip_restrictions_deny(self, resource_group):
             JMESPathCheck('length(@)', 0),
         ])
 
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="northeurope")
+    def test_containerapp_cors_policy(self, resource_group):
```

Comment:
Please run the following command to ceate recording file:
`azdev test test_containerapp_cors_policy --live`